scripts/dashboard-importer/README.md

```diff
@@ -0,0 +1,184 @@
+In the directory with the file(s) to be uploaded, it also creates `upload_<time>.txt` file that records the names and URLs of the uploaded files
+
+## Troubleshooting
+Common troubleshooting cases for the Cloud Monitoring Dashboard Importer
+
+### gcloud Related Issues
+
+#### gcloud: command not found
+
+You need to install gcloud. See the
+[gcloud Installation Docs](https://cloud.google.com/sdk/docs/install)
+
+#### Problem refreshing current auth tokens
+
+You need to run `gcloud auth login`. See the [docs](https://cloud.google.com/sdk/gcloud/reference/auth/login) on obtaining access credentials.
+
+### Can't find output files
+
+Converted JSON files are written to the `reports/<date>/<time>/` directory
+of the importer.
+
+In the same directory, you will also find:
+
+-   `report.json` - contains information regarding the conversion such warnings
+    and errors that occurred
+-   `upload_<time>.txt` - record of the generated cloud monitoring urls from uploading
+
+### Conversion Issues
+
+#### Dashboard has tiles with no data coming in
+
+We recommend the following steps for debugging tiles with no data:
+
+1.  Confirm that the corresponding chart has data in Grafana as well. If your chart does not have data in Grafana, the chart in the converted dashboard most likely will not have data either.
+1.  Confirm all templating variables referenced in the queries exist in the Cloud Monitoring
+    dashboard. There are some templating variables that the importer doesn't handle. See the [missing templating variable section](#templating-variables-are-missing)
+1.  Confirm the metric(s) being referenced exist in Cloud Monitoring and are not
+    inactive. You can do so by searching for the query's metric in the
+    [Metrics Explorer](https://console.cloud.google.com/monitoring/metrics-explorer). If the metric doesn't exist, you will need to find an alternative metric.
+1.  Confirm that the label matchers are valid.
+    To validate this, first look at the query and determine if there are any hard equality matchers. Then look up the metric used in the
+    [Metrics Explorer](https://console.cloud.google.com/monitoring/metrics-explorer).
+    and ensure both the label key and value exist. If they do not, you will need to
+    manually resolve this by either removing or modifying the matcher.
+
+#### Tiles are missing
+
+Tiles that are under the height of 2 or are missing x, y, h, w positional
+attributes are omitted. You will need to add/modify those attributes and rerun
+the importer.
+
+#### Tiles are not aligned / out of order
+
+You may need to drag and resize some tiles for some cases of misalignment.
+
+For severe cases of misalignment that cannot be remedied by any form of
+repositioning and resizing, please file an issue with details under this
+[Github repository](https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/tree/master/scripts/dashboard-importer).
```

Comment:
FWIW, https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples is the Github repository and this is a subdirectory of that; and you leave issues at the repository level. Either link to `monitoring-dashboard-samples` or just remove the link entirely (because anyone reading the README is already here anyway).

Reply:
Removed the link!